lib/src/summarize.dart

```diff
@@ -4,63 +4,301 @@
 
 library dart_pad.summarize;
 
+import 'package:crypto/crypto.dart';
+
 import '../services/dartservices.dart';
 
 /// Instances of this class take string input of dart code as well as an analysis result,
 /// and output a text description ofthe code's size, packages, and other useful information.
 class Summarizer {
   _SummarizeToken storage;
-
+  String dart;
+  String css;
+  String html;
   bool resultsPresent;
+  int randomizer;
+  AnalysisResults analysis;
+
+  static Map<String, List<int>> cuttoffs = {
+    'size': [8, 30, 1000], //0-7 = small, 8 - 29 = decent, 29+ = gigantic
+    'errorCount': [1, 5, 100]
+  };
+
+  static Map<String, String> codeKeyWords = {
+    'await': 'await',
+    'async': 'async',
+    'rpc': 'RESTful serverside app'
+  };
 
-  Summarizer(String input, [AnalysisResults analysis]) {
-    if (input == null) throw new ArgumentError("Input can't be null.");
+  static Map<String, String> additionKeyWords = {
+    'pirate': 'pirates',
+    'bird': 'birds',
+    'llama': 'llamas',
+    'dog': 'dogs'
+  };
+
+  static Map<String, List<String>> categories = {
+    /// This [size] [codeQuantifier] contains [error] errors and warnings.
+    'size-2': [
+      'gigantic',
+      'Jupiterian sized',
+      'immense',
+      'massive',
+      'enormous',
+      'huge',
+      'epic',
+      'humongous'
+    ],
+    'size-1': [
+      'decently sized',
+      'exceptional',
+      'awesome',
+      'amazing',
+      'visionary',
+      'legendary'
+    ],
+    'size-0': ['itty-bitty', 'miniature', 'tiny', 'pint-sized'],
+    'compiledQuantifier': ['DART program', 'pad'],
+    'failedQuantifier': [
+      'assemblage of characters',
+      'series of strings',
+      'grouping of letters'
+    ],
+    'errorCount-2': [
+      'many',
+      'a motherload of',
```

Comment:
typo: motherlode, I think the etymology is this: http://en.wikipedia.org/wiki/Mother_lode